src/index.html

```diff
@@ -5,7 +5,7 @@
 
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href ="styles.css">
+  <link rel="stylesheet" href ="main.scss">
```

Comment:
You don't need this line anymore since this file is being imported in `app.component.scss`.